overlays/template/kustomization.yaml

```diff
@@ -21,7 +21,7 @@ images:
   - name: quay.io/rh_integration/3scale-fuse-camel-proxy
     newName: quay.io/rh_integration/3scale-fuse-camel-proxy
     newTag: latest
-  - name: quay.io/rh_integration/tinyproxy
+  - name: ghcr.io/3scale-qe/tinyproxy
     newName: quay.io/rh_integration/tinyproxy
     newTag: eloy
```

Comment:
please change tag as well

Reply:
I think the change is correct. This is an example how to kustomize deployment. And upstream `ghcr.io/3scale-qe/tinyproxy:latest` is kustomized to `quay.io/rh_integration/tinyproxy:eloy`. `newName` as well as `newTag` should be set as they are